app/models/import/row.rb

```diff
@@ -0,0 +1,36 @@
+module Import
+  class Row
+    KEYS = [
+      :reported_on, :in_care, :referrals, :discharges,
+      :funded, :reserve, :activated, :unavailable
+    ].freeze
+
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def call
+      capacity.assign_attributes(attributes)
```

Comment:
The lack of parentheses on method calls keeps throwing me, but once I get past that, I really like the way this works.  It's super readable.